tests/robot-tests/tests/libs/common.robot

```diff
@@ -718,3 +745,17 @@ check that variable is not empty
 user waits until table tool wizard step is available
     [Arguments]    ${table_tool_step_title}    ${wait}=${timeout}
     user waits until element is visible    xpath://h2|h3//*[contains(text(),"${table_tool_step_title}")]    ${wait}
+
+get webelement
```

Comment:
SeleniumLibrary has a `get webelement` keyword - so I'm not sure how this interacts with the selenium library keyword?

Reply:
Ooh I didn't realise - I'll check it out and I'll also rename this to avoid shadowing

Reply:
Renamed